native/lambda_game_engine/src/player.rs

```diff
@@ -28,6 +28,7 @@ pub struct Player {
     pub size: u64,
     pub speed: u64,
     pub action_duration_ms: u64,
+    pub skills_to_execute: Vec<String>,
```

Comment:
Don't see why the player should have this, `Effect` already holds this information and when that effect triggers based on its `Periodic` setting they should be executed

Reply:
Discussed offline, this is an insane can of worms and no clear solution due to the need to have the `Game` modify itself when activating a skill, but when doing an effect on a player you don't have that information and passing it would break a lot of responsibility boundaries in the code

Long story short, we decided that this is a good middle ground until we figure a better approach